ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -370,7 +370,7 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         }
 
         label.text = text
-        trailingImageView.image = accessoryType.image
+        trailingImageView.image = interactive ? accessoryType.image : nil
         trailingImageView.isHidden = trailingImageView.image == nil
```

Comment:
Would it make more sense to move this logic up into the `if interactive {` on line 362?

Reply:
Six of one, half a dozen of the other, if you ask me, but yeah, that's fine.